src/Pages/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabID } = useParams();
+  const currentTab = tabs.find(tab => tab.id === tabID);
```

Comment:
```suggestion
  const tabContent = tabs
    .find(tab => tab.id === tabID)?.content || 'Please select a tab';
```

You can get a content here instead of doing it inside JSX